cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -179,6 +180,11 @@ class DownloaderCommand : OrtCommand(
                 "result to limit downloads to. If not specified, all packages are downloaded."
     ).split(",")
 
+    private val skipExcluded by option(
+        "--skip-excluded",
+        help = "Do not download excluded projects or packages. Works only with the '--ort-file' parameter."
+    ).flag("--include-excluded", default = true)
```

Comment:
Setting this to true would make this a breaking change.

I personally think that defaulting to omitting excluded projects is good, other tools (scanner) default to include excluded projects.
So we might want to default to `false` in this case as well, for consistency.

What do others think?

Reply:
@MarcelBochtler I do agree, I would prefer `true` as default, but for consistency it would be better to use `false`.
@bs-ondem Please use a name for the first argument too.

Reply:
Why do we need `--include-excluded` to invert this after all? The other commands that support `--skip-excluded` (`AdvisorCommand`, `ScannerCommand`) don't have it, either. In any case, I'd vote for consistency here. Also "include exclude" sounds a bit odd. Maybe `--contain-excluded` or `--download-excluded`?

Reply:
That's right, if the option has a default value the inverted flag is not required.

Reply:
> Why do we need --include-excluded to invert this after all?

If the default value for a flag is set to true, you need a `secondaryName` to set it to false. Otherwise, you can never set it to false. By setting the default value to  `false` now, the `secondaryName` is not needed anymore.

Reply:
BTW, in the context of our general effort to reduce CLI options in favor of config options, should this really become a new CLI option? Like, what's the use-case exactly / how often would you invoke the downloader with different values for this option?

Reply:
The generated source code bundle (`--archive-all` option) contains sources for e.g. test related packages, which are actually excluded. With this option, we can prevent adding sources of excluded packages in it.

Reply:
So, this means you *always* want to run the downloader CLI with `--skip-excluded`, and from that perspective it *could* become a config option. But I'm fine with not adressing this now as part of this PR, as it would require some alignment with `AdvisorCommand` and `ScannerCommand`, too.